group_vars/rhel6

```diff
@@ -1,24 +1,37 @@
-repositories:
-  - name: epel
+all_repositories:
+  latest:
+    name: rhel6-latest
+    baseurl: "{{ rhel_base }}/$basearch/os/"
+  optional:
+    name: rhel6-optional
+    baseurl: "{{ rhel_base }}/$basearch/optional/os/"
+  extras:
+    name: rhel6-extras
+    baseurl: "{{ rhel_base }}/$basearch/extras/os/"
+  epel:
+    name: epel
     mirrorlist: "{{ fedora_mirrors }}repo=epel-6"
-  - name: epel-testing
-    mirrorlist: "{{ fedora_mirrors }}repo=testing-epel6"
-  - name: epel
-    mirrorlist: "{{ fedora_mirrors }}repo=epel-6"
-    includepkgs:
-      - python-html5lib
-      - python-beautifulsoup4
-      - python-blessings
-      - python-html5lib # Duplication is present in old version - typo?
-  - name: rhel6-latest
-    baseurl: "{{ rhel_base }}/latest-RHEL-6/compose/Server/x86_64/os/"
-  - name: rhel6-optional
-    baseurl: "{{ rhel_base }}/latest-RHEL-6/compose/Server/optional/x86_64/os/"
-  - name: rhel6-extras
-    baseurl: "{{ rhel_base }}/latest-EXTRAS-6-RHEL-6/compose/Server/x86_64/os/"
-  - name: rhscl-python27
-    baseurl: https://www.softwarecollections.org/repos/rhscl/python27/epel-6-x86_64/
+  rhscl_python27:
+    name: rhscl-python27
+    baseurl: https://www.softwarecollections.org/repos/rhscl/python27/epel-6-$basearch/
 
 repository_downloads:
```

Comment:
As far as I know consvr is still in use according to the previous project (targets/repos.py).  repository_downloads is not used elsewhere, I'll remove that.